content/en/continuous_integration/setup_tests/dotnet.md

```diff
@@ -141,6 +143,33 @@ To use the custom instrumentation in your .NET application:
 
 For more information about how to add spans and tags for custom instrumentation, see the [.NET Custom Instrumentation documentation][4].
 
+## Agentless (Beta)
+
+To instrument your test suite without requiring an agent, prefix your test command with `dd-trace ci run`, providing the [Datadog API key][6] as the `--api-key`, the name of the service or library under test as the `--dd-service` parameter, and the environment where tests are being run (for example, `local` when running tests on a developer workstation, or `ci` when running them on a CI provider) as the `--dd-env` parameter. For example:
+
+{{< code-block lang="bash" >}}
+dd-trace ci run --api-key <API KEY> --dd-service=my-dotnet-app --dd-env=ci -- dotnet test
```

Comment:
The wording in this section seems to recommend using the `--api-key` param, but I think we should be recommending the env var to avoid persisting the api key in the terminal's history.

Reply:
Changed in [f24b1da](https://github.com/DataDog/documentation/pull/13609/commits/f24b1daab93b8669fa6e3a73f36feffbef732c11)